### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by shruti-sen2004**
- **What kind of change does this PR introduce?** 
-  Updated the contributing.md file . I have chandes the worjflow section and made it easy to understand.

- **Why was this change needed?** 
-  It will help anyone who is a first time contributor

- **Other information**:
- No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,8 +44,10 @@ To run unit tests from the root of the repository, execute:
 python -m pytest
 ```
 
-### Workflow:
-Fork the repository, make your changes on your forked version, and then submit those changes as a pull request.
+### Workflow📈 :
```
Comment (dartpain) [inline CONTRIBUTING.md:47]: Space between text and emoji

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,8 +44,10 @@ To run unit tests from the root of the repository, execute:
...
+- Fork the repository <br>
```
Comment (dartpain) [inline CONTRIBUTING.md:48]: no need for <br>

**Review by dartpain — CHANGES_REQUESTED** [review]
Include space

and remove <br>

Thank you!

File: CONTRIBUTING.md (hunk 1 above)
Comment (shruti-sen2004) [reply CONTRIBUTING.md]: okay

File: CONTRIBUTING.md (hunk 2 above)
Comment (shruti-sen2004) [reply CONTRIBUTING.md:48]: okay

**Comment by shruti-sen2004** [comment]
I have made the necessary changes .

**Comment by dartpain** [comment]
The <br> is still present

**Comment by shruti-sen2004** [comment]
I have again updated it. Can you pleas check.

**Comment by dartpain** [comment]
Sorry this is markdown, please bring back the work the

I mean that I only want you to remove < br > tag

**Comment by shruti-sen2004** [comment]
Okay I think ...now i got it. :)

**Comment by dartpain** [comment]
There is one more br tag on line 50
thank you

**Comment by shruti-sen2004** [comment]
I have updated the file......Can you please check?

**Comment by shruti-sen2004** [comment]
Can you please merge it.
Thank you

**Review by dartpain — APPROVED** [review]

**Pull request merged**